src/Components/Aspire.Elastic.Clients.Elasticsearch/Aspire.Elastic.Clients.Elasticsearch.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>$(ComponentCommonPackageTags) Elasticsearch</PackageTags>
+    <Description>A Elasticsearch client that integrates with Aspire, including health checks, logging, and telemetry.</Description>
```

Comment:
Is there an icon/logo we can use for these 2 new packages?

Reply:
Also, can you update the following files:

* https://github.com/dotnet/aspire/blob/main/src/Components/Aspire_Components_Progress.md
* https://github.com/dotnet/aspire/blob/main/src/Components/Telemetry.md